extensions/extensions-helm/aissemble-spark-operator-chart/values.template.yaml

```diff
@@ -30,11 +30,6 @@ spark-operator:
     fsGroup: 1000
     fsGroupChangePolicy: "OnRootMismatch"
 
-  rbac:
-    # -- Create and use RBAC `ClusterRole` resources
-    # -- Set to false in order to enable overriding with our own RBAC template
-    createClusterRole: false
```

Comment:
Q: Assuming this defaults to `true` now?

Reply:
Indeed